include/dem/visualization.h

```diff
@@ -133,7 +129,7 @@ class Visualization : public dealii::DataOutInterface<0, dim>
   /**
    * Number of properties that are written in output files
    */
-  const unsigned int number_of_properties_to_write = 16;
+  const unsigned int number_of_properties_to_write = 9;
```

Comment:
Why is this hardcoded? Couldnt we just assume that we write all properties using the dem_parameters enum?